src/codeu/chat/client/core/Context.java

```diff
@@ -48,9 +48,9 @@ public UserContext create(String name) {
     }
     return users;
   }
-  
+
+  // view is typecasted in order to access getInfo()
   public ServerInfo getInfo() {
-	View infoView = (View)view;
-	return infoView.getInfo();
-  }
+    View infoView = (View)view;
```

Comment:
Same comment as for Antonio - I think it is better to avoid the casting and define `getInfo()` in the base class `BasicView`. If this has already been taken care in a future commit, ignore this comment.